src/openagi/actions/tools/yahoo_finance.py

```diff
@@ -72,4 +72,7 @@ def execute(self) -> str:
 				return f"Invalid info_type: {self.info_type}. Supported types are: summary, price, history, info"
 				
 		except Exception as e:
-			return f"Error fetching data for {self.symbol}: {str(e)}"
\ No newline at end of file
+			return f"Error fetching data for {self.symbol}: {str(e)}"
+
+	async def async_execute(self):
+		return self.execute()
```

Comment:
formatting

Reply:
Checked in vs code and github.dev but shows correct indentation